.env

```diff
@@ -0,0 +1 @@
+REACT_APP_BACKEND_URL="http://192.168.10.1:4877"
```

Comment:
Good catch. Justin had addressed this already by using the window.location.hostname instead of hardcoding it but it got reverted when I tried to move the setting into an environment variable. I've addressed it now by using the env variable if it's defined so I can work with my local dev environment which doesn't run on the same IP as the backend, or else defaulting to the hostname.